pkg/kritis/cron/cron.go

```diff
@@ -19,32 +19,96 @@ package cron
 import (
 	"context"
 	"time"
+
+	"github.com/grafeas/kritis/pkg/kritis/apis/kritis/v1beta1"
+	"github.com/grafeas/kritis/pkg/kritis/crd/securitypolicy"
+	"github.com/grafeas/kritis/pkg/kritis/metadata"
+	"github.com/grafeas/kritis/pkg/kritis/metadata/containeranalysis"
+	"github.com/grafeas/kritis/pkg/kritis/violation"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/sirupsen/logrus"
 )
 
 // For testing
 var (
-	podChecker    = CheckPods
 	checkInterval = 1 * time.Minute
+	podChecker    = CheckPods
+)
+
+// For testing.
+type podLister func(string) (*v1.PodList, error)
+type violationChecker func(string, v1beta1.ImageSecurityPolicy) ([]metadata.Vulnerability, error)
+type violationHandler func(string, *v1.Pod, []metadata.Vulnerability) error
+
+type Config struct {
+	PodLister         podLister
+	ViolationChecker  violationChecker
+	ViolationStrategy violation.Strategy
+}
+
+var (
+	defaultViolationStrategy = &violation.LoggingStrategy{}
 )
 
+func NewCronConfig(cs *kubernetes.Clientset, ca containeranalysis.ContainerAnalysis) *Config {
+	pl := func(ns string) (*v1.PodList, error) {
+		return cs.CoreV1().Pods(ns).List(metav1.ListOptions{})
```

Comment:
We have this [function](https://github.com/grafeas/kritis/blob/master/pkg/kritis/pods/pods.go#L36) `Pods(namespace string) ([]corev1.Pod, error)` in the pods package, which we could use here?